modules/renter/repair.go

```diff
@@ -17,7 +17,7 @@ import (
 
 // TODO: Move to a consts file.
 const uploadFailureCooldown = time.Hour
-const minPiecesRepair = 6
+const minPiecesRepair = 5
```

Comment:
This optimizes for network performance slightly at the cost of encryption overhead. But encryption overhead is not very high at the moment, network performance still seems to be the bottleneck on the tail end of uploading.